homeassistant/components/shelly/entity.py

```diff
@@ -321,12 +321,13 @@ class RestEntityDescription(EntityDescription):
 class ShellyBlockEntity(CoordinatorEntity[ShellyBlockCoordinator]):
     """Helper class to represent a block entity."""
 
+    _attr_should_poll = False
+
```

Comment:
CoordinatorEntity's already have `_attr_should_poll = False`
```suggestion
```

Reply:
Oh you're right, sometimes I'm blind 😉

Reply:
I mean I also did not know until 2 weeks ago, but if you know you know, so now you do as well :)

Reply:
All Shelly entity classes inherit indirectly from `BaseCoordinatorEntity` and this class returns `False`:
https://github.com/home-assistant/core/blob/99a76ef4e6d6ca463ccad564ff6c79118cab8178/homeassistant/helpers/update_coordinator.py#L452-L455

So, we don't need `_attr_should_poll = False` in any entity class at all.